lib/moribus/extensions/has_current_extension.rb

```diff
@@ -11,14 +11,12 @@ def remove_target!(*)
         else
           # Use custom update to avoid running ActiveRecord optimistic locking
           # and to avoid updating lock_version column:
-          klass          = target.class
-          is_current_col = klass.columns.detect { |c| c.name == "is_current" }
-          id_column      = klass.columns.detect { |c| c.name == klass.primary_key }
+          klass = target.class
 
           sql =  "UPDATE #{klass.quoted_table_name} " \
-                 "SET \"is_current\" = #{klass.quote_value(false, is_current_col)} "
+                 "SET \"is_current\" = #{klass.connection.quote(false)} "
```

Comment:
Can you provide a little more background on this change?

Reply:
```
NoMethodError: 
    undefined method `quote_value' for #<Class:0x007fc4aaf01ba0>
```
The last version when the `quote_value` method appeared in `activerecord` repo was 3.1 https://github.com/rails/rails/blob/3-1-stable/activerecord/lib/active_record/base.rb#L850 So I replaced it with rails 5.2 method `quote` https://github.com/rails/rails/blob/aeba121a83965d242ed6d7fd46e9c166079a3230/activerecord/lib/active_record/connection_adapters/abstract/quoting.rb#L11

Reply:
Then how was it working in 4.2?

Reply:
Then how was it working in 4.2?